content/applications/hr/fleet/models.rst

```diff
@@ -0,0 +1,194 @@
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: this value is automatically calculated based on Belgian laws and regulations,
+  and cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions`
+  field in the :guilabel:`Engine` section of the vehicle form.
+
+.. important::
+   Modifying the :guilabel:`CO2 Emissions` field will adjust the value in the :guilabel:`CO2 fee`
+   field.
+
+- :guilabel:`Cost (Depreciated)`: enter the monthly cost for the vehicle, which appears in the
+  salary configurator that is available to a future employee when they are offered a job position.
+  This value impacts the gross and net salary of the employee who is assigned to the vehicle. This
+  figure is depreciated over time according to local tax laws. The :guilabel:`Cost (Depreciated)`
+  does not depreciate automatically on the *vehicle model*, it only depreciates based on the
+  *contract* linked to a specific vehicle and not on the general model.
+- :guilabel:`Total Cost (Depreciated)`: this value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+Engine
+~~~~~~
+
+- :guilabel:`Fuel Type`: select the type of fuel the vehicle uses from the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: enter the average carbon dioxide emissions the vehicle produces in
+  grams per kilometer (g/km). This information is provided by the car manufacturer.
+- :guilabel:`CO2 Standard`: enter the standard amount of carbon dioxide in grams per kilometer
+  (g/km) for a similar sized vehicle.
+- :guilabel:`Transmission`: select the transmission type from the drop-down menu, either
+  :guilabel:`Manual` or :guilabel:`Automatic`.
+- :guilabel:`Power`: if the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: enter the amount that is taxed based on the size of the vehicle's
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: this field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
+
+Vendors tab
+-----------
+
+Vehicle specific vendors, such as car dealerships, are not listed separately from other vendors. The
+vendors that a vehicle can be purchased from also appear in the list of vendors used by the
+*Purchase* app.
```

Comment:
Totally optional suggestion! I wasn't really sure what this feature was for, and i wanted to try to make it more clear. Feel free to ignore/make changes if it's wrong!
```suggestion
Specify the vendor(s) a vehicle can be purchased from in this tab. With proper setup, requests for quotations for vehicles can be easily created through Odoo's *Purchase* app.
```